docs/docsite/rst/dev_guide/platforms/vmware_guidelines.rst

```diff
@@ -0,0 +1,180 @@
+.. _VMware_module_development:
+
+****************************************
+Guidelines for VMware module development
+****************************************
+
+The VMware modules and these guidelines are maintained by the VMware Working Group. For
+further information see the team community page: `https://github.com/ansible/community/wiki/VMware`.
+
+.. contents::
+   :local:
+
+Testing with vcsim
+==================
+
+Most of the existing modules are covered by functional tests. The tests are located in the :file:`test/integration/targets/`.
+
+By default, the tests run against a vCenter API simulator called `govcsim <https://github.com/vmware/govmomi/tree/master/vcsim>`. ``ansible-test`` will automatically pull a `govcsim container <https://quay.io/repository/ansible/vcenter-test-container>` and use it to set-up the test environment.
+
+You can trigger the test of a module manually with the ``ansible-test`` command. For example, to trigger ``vcenter_folder`` tests:
+
+.. code-block:: shell
+
+    source hacking/env-setup
+    ansible-test integration --python 3.7 vcenter_folder
+
+``vcsim`` is handy because it's much more fast that than a regular test environment. However, it does not
+support all the ESXi or vcenter features.
+
+Testing with your own infrastructure
+====================================
+
+You can also target a regular VMware environment. This paragraph explains step by step how you can run the test-suite yourself.
+
+Requirements
+------------
+
+- 2 ESXi hosts (6.5 or 6.7)
+   - with 2 NIC, the second ones should be available for the test
+- a VCSA host
+- a NFS server
+
+You will need to install a couple of Python dependencies, like `pyvmomi <https://github.com/vmware/pyvmomi/tree/master/pyVmomi>` and `requests <https://2.python-requests.org/en/master/>`.
```

Comment:
I'd change this to be part of the list of Requirements:
- an NFS server
- Python dependencies
   - pyvmomi
   - requests

Reply:
I agree, it sounds more logical.